src/electionguard_verify/verify.py

```diff
@@ -67,3 +66,28 @@ def verify_decryption(
                     )
 
     return Verification(True, message=None)
+
+
+def verify_aggregation(
+    submitted_ballots: List[SubmittedBallot],
+    tally: CiphertextTally,
+    internal_manifest: InternalManifest,
```

Comment:
I should have caught this on first review but this should take the `Manifest` not the `InternalManifest`.